Automattic/vip-go-mu-plugins
# Update ElasticPress submodule to latest

## Description

Updates the ElasticPress submodule to the latest revision, which is just a new filter to bail early from the indexing process.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. `git submodule update --init --recursive`
1. Test various search functionality - `?s=test`, `wp vip-search health validate-counts`, etc
